ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -639,9 +639,9 @@ private void writeLines(AmplExportContext context, TableFormatter formatter) thr
                         .writeCell(-1) // no ratio tap changer
                         .writeCell(-1) // no phase tap changer
                         .writeCell(t1.getP())
-                        .writeCell(-tl.getHalf1().getXnodeP()) // xnode node flow side 1
+                        .writeCell(-tl.getBoundaryPoint(Branch.Side.ONE).getP()) // xnode node flow side 1
```

Comment:
I am not sure to like `BoundaryPoint` name choice indeed. We have no `Point` in IIDM: we have `node`, `bus` and `terminal`. When you do `getBoundaryPoint(Branch.Side.ONE).getP()` for me you refer to a terminal and it will be clearer to do `getHalf1().getBoundaryTerminal().getP()`.

Reply:
I don't like using the term `Terminal` when it is not *really* a `Terminal` object... It is a bit confusing, for me it is not a node, bus or terminal and this is why I created another term.

Reply:
Another proposal:
`tieLine.getHalf1().getOtherSide().getP()`
`danglingLine.getOtherSide().getP()`